app/views/admin/conferences/edit.html.haml

```diff
@@ -24,5 +24,5 @@
         = f.input :start_hour, :input_html => {size: 2, type: 'number', min: 0, max: 23}
         = f.input :end_hour, :input_html => {size: 2, type: 'number', min: 1, max: 24}
       = f.inputs name: "Registrations" do
-        = f.input :registration_limit, as: :number, in: 0..9999, hint: "Limit the number of registrations to the conference (0 no limit)"
+        = f.input :registration_limit, as: :number, in: 0..9999, hint: "Limit the number of registrations to the conference (0 no limit). You already have #{@conference.registrations.count} registrations."
```

Comment:
I'd rather say 'you currently have...' instead of you 'you already have' because it sounds right even if it is 0 registrations, and we are not checking if there are actually any registrations. 

Also if there is 1 registration, it should read 'you have 1 registration', not '1 registrations'.